microsoft-edge/devtools-guide-chromium/css/inspect.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Learn how to use Microsoft Edge DevTools to analyze HTML pages using the Inspect tool 
+title: Inspect Tool in Microsoft Edge DevTools
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 12/16/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+
+# Analyze HTML pages using the Inspect tool 
+
+This article shows you how to use the **Inspect** tool to preview information about and select an element from the current document. You can try it out yourself by checking the [Inspect Demo Page](https://microsoftedge.github.io/DevToolsSamples/inspector/inspector-demo.html).
+
+The **Inspect** tool button (![Inspect](../media/inspect-icon.msft.png)) is in the upper-left corner of DevTools.  When you select the **Inspect** tool button, the button turns blue, indicating that the **Inspect** tool is active.
+
+:::image type="complex" source="images/inspect-tool-button.msft.png" alt-text="The inspect tool button on the top right of the Developer Tools" lightbox="images/inspect-tool-button.msft.png":::
+   The inspect tool button on the top right of the Developer Tools
+:::image-end:::
+
+
+Alternatively you press `Control`+`Shift`+`C` (Windows, Linux) or `Command`+`Shift`+`C` (macOS) to activate it. 
+
+## Getting element information from the Inspect overlay
+
+When the **Inspect** tool is active, hovering over any element on the rendered webpage displays the **Inspect** overlay. This overlay displays general and accessibility information about that element.  
+
+:::image type="complex" source="images/inspect-tool-padding-margin.msft.png" alt-text="The inspector overlay showing the information of a heading in the browser":::
+   The inspector overlay showing the information of a heading in the browser
+:::image-end:::
+
+As you move around the screen, the source view showing the DOM of the document updates and highlights the element you are hovering on in the document tree. 
+
+The **Inspect** overlay shows the following information, depending on the type of element and the styles applied to it:
+
+* The name of the element
+* Its dimensions in pixels
+* Its color as a hexidecimal value and a color swatch
+* Its font settings
+* Its margin and padding in pixels
+
+If the element has been positioned using CSS grid or CSS flexbox you get different icons displayed next to its name.
```

Comment:
Minor rephrasing proposal: `If the element is positioned using CSS grid or CSS flexbox, a different icon is displayed next to the element's name`.